docs/troubleshooting.md

```diff
@@ -71,17 +69,7 @@ scutil --dns
 bin/setup
 ```
 
-## Troubleshoot: Installation
-
-### Diagnose common issues when setting up GOV.UK Docker
-
-Run the following command in `~/govuk/govuk-docker`. Since this script makes use of Ruby Gems, you will need to [install some additional dependencies](../CONTRIBUTING.md#testing) in order to do this.
-
-```
-bin/doctor
-```
```

Comment:
May be worth linking to https://github.com/alphagov/govuk-docker/issues/435 in your commit, as I wouldn't have known why this section was removed if I hadn't already read your issue..

Reply:
Ah, so I was wondering whether to be explicit about this in the commit: the section still exists at the top - it's just de-duplicated. Depending on what happens with #435, we may yet remove or edit it, but I agree that would be premature here.

Is that OK?

Reply:
Ah, didn't spot that. Makes sense. Thanks